templates/pages/errors/403.html

```diff
@@ -3,14 +3,14 @@
 {{> components/common/breadcrumbs breadcrumbs=breadcrumbs}}
```

Comment:
@aleksandraworhacz please enclose the page content in a `<div>` like in this other page: https://github.com/xfiveco/xcorner/blob/tailwind/templates/pages/category.html#L18-L20 for consistency sake :) 

Same for 404.html

Thank you.